From 3637621b2fe4d263590f221d3160aee63d00f65f Mon Sep 17 00:00:00 2001
From: Bart Oostveen <bart.oos.2006@gmail.com>
Date: Mon, 16 Feb 2026 15:16:02 +0100
Subject: [PATCH] fix: go 1.25.5

---
 go.mod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 83a4ab9..acf5f16 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/liminalpurple/matrix-stickerbook
 
-go 1.26.0
+go 1.25.5
 
 require (
 	github.com/anthropics/anthropic-sdk-go v1.22.1
-- 
2.52.0

